code/modules/power/power.dm

```diff
@@ -291,7 +291,7 @@
 		var/mob/living/carbon/human/H = M
 		if(H.gloves)
 			var/obj/item/clothing/gloves/G = H.gloves
-			if(G.siemens_coefficient == 0)
+			if(G.siemens_coefficient == 0 && !istype(G, /obj/item/clothing/gloves/color/yellow/sprayon)) //sorry for snowflake
```

Comment:
It seems like this would cause disparity between these and insuls, is this the case/intended?

Reply:
So essentially what the !istype is doing is executing electrocute_act anyway if the gloves are spray-ons, since otherwise it never calls electrocute_act and thus never decrements shocks_remaining, leading to the gloves simply acting like non-removable insuls, instead of limited use ones.

Reply:
this is way too snowflake